.circleci/config.yml

```diff
@@ -90,17 +90,11 @@ jobs:
             bash ./codecov -v -X s3 -c -D "./build/out" -J "AEPEdge"
 
       # verify XCFramework archive builds
+      # ci-archive updates Pods so run this step last
       - run:
           name: Build XCFramework
           command: |
-            if [ "${CIRCLE_BRANCH}" == "main" ]; then
+            if [[ "${CIRCLE_BRANCH}" == "main" || "${CIRCLE_BRANCH}" == "staging" ]]; then
               make ci-archive
             fi
 
-      # verify demo app builds
-      - run:
-          name: Build Demo App
-          command: |
-            if [ "${CIRCLE_BRANCH}" == "main" ]; then
-              make build-app
-            fi
```

Comment:
Should we keep this code around and comment it out until we get it fixed?

Reply:
As it's in source control we have access to the code if needed. I'd prefer to keep it clean and remove the code.